<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskPanel</class>
 <widget class="QWidget" name="TaskPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create X-Ray machine</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="emitter_group">
     <property name="toolTip">
      <string>The X-Ray emitter options, including the power spectrum (which is renormalized, so do not worry about the units) and the bulb size.</string>
     </property>
     <property name="title">
      <string>XRay emitter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="freqs_group">
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="min_freq">
           <property name="text">
            <string notr="true">1 keV</string>
           </property>
           <property name="unit" stdset="0">
            <string notr="true">eV</string>
           </property>
           <property name="quantity" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sep_freq">
           <property name="text">
            <string notr="true">-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="max_freq">
           <property name="text">
            <string notr="true">10 keV</string>
           </property>
           <property name="unit" stdset="0">
            <string notr="true">eV</string>
           </property>
           <property name="quantity" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="preset_group">
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="preset_label">
           <property name="text">
            <string>Load preset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="preset">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Medical shoft X-Rays</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Crystallography</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Medical hard X-Rays</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Airport security</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Metallography</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="spectrum_group">
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="power_group">
         <item>
          <widget class="QSlider" name="spectrum_01">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_02">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_03">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_04">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_05">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_06">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_07">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_08">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_09">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_10">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_11">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_12">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_13">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_14">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_15">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_16">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_17">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_18">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_19">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="spectrum_20">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="freq_n_group">
        <property name="toolTip">
         <string>The number of samples to take fro the spectrum. The number of images per projection to be rendered is equal to the number of samples divided by 3. Thus the larger the number of samples, the longer would it take to compute. Increasing the number of samples would be very beneficial in the presence of metals.</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="freqs_n_label">
           <property name="text">
            <string>Number of freq. samples</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="freqs_n">
           <property name="minimum">
            <number>3</number>
           </property>
           <property name="maximum">
            <number>21</number>
           </property>
           <property name="singleStep">
            <number>3</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QGroupBox" name="projection_type_group">
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="projection_label">
           <property name="text">
            <string>Emitter type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="projection">
           <property name="toolTip">
            <string>For the time being just Parallel beams (radiographies and old-school CTs) is implemented</string>
           </property>
           <item>
            <property name="text">
             <string>Parallel beams</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="chamber_group">
     <property name="toolTip">
      <string>The scanning chamber dimensions. At first glance just the objects fitting inside will be correctly scanned. However, in some cases it should be accepted that the scanning body is larger than the area covered by the X-Rays, that would be the case in synchrotron-based radiation.</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="radius_label">
        <property name="text">
         <string>radius</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="height_label">
        <property name="text">
         <string>height</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="radius">
        <property name="text">
         <string notr="true">50 mm</string>
        </property>
        <property name="unit" stdset="0">
         <string notr="true">m</string>
        </property>
        <property name="quantity" stdset="0">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="height">
        <property name="text">
         <string notr="true">50 mm</string>
        </property>
        <property name="unit" stdset="0">
         <string notr="true">m</string>
        </property>
        <property name="quantity" stdset="0">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="distance_label">
        <property name="text">
         <string>distance</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="distance">
        <property name="text">
         <string notr="true">50 cm</string>
        </property>
        <property name="unit" stdset="0">
         <string notr="true">m</string>
        </property>
        <property name="quantity" stdset="0">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="sensor_group">
     <property name="title">
      <string>Sensor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QGroupBox" name="resolution_group">
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="resolution_label">
           <property name="text">
            <string>Resolution</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="resolution_x">
           <property name="minimum">
            <number>16</number>
           </property>
           <property name="maximum">
            <number>32768</number>
           </property>
           <property name="value">
            <number>1024</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="resolution_y">
           <property name="minimum">
            <number>16</number>
           </property>
           <property name="maximum">
            <number>32768</number>
           </property>
           <property name="value">
            <number>1024</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
